exasol_integration_test_docker_environment/lib/api/build_test_container.py

```diff
@@ -10,16 +12,16 @@ def build_test_container(
         force_rebuild: bool = False,
         force_rebuild_from: Tuple[str, ...] = tuple(),
         force_pull: bool = False,
-        output_directory: str = ".build_output",
+        output_directory: str = DEFAULT_OUTPUT_DIRECTORY,
         temporary_base_directory: str = "/tmp",
         log_build_context_content: bool = False,
         cache_directory: Optional[str] = None,
         build_name: Optional[str] = None,
-        source_docker_repository_name: str = 'exasol/script-language-container',
+        source_docker_repository_name: str = DOCKER_REPOSITORY_NAME,
```

Comment:
maybe rename to DEFAULT_DOCKER_REPOSITORY_NAME

Reply:
Done